infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -142,6 +150,11 @@ protected void processRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRo
         executor.submit(() -> getHandler().handle(msg, ctx, r));
     }
 
+    @Override
+    protected void processRequest(RequestMsg req, ChannelHandlerContext ctx, IServerRouter r) {
+        executor.submit(() -> getHandlerMethods().handle(req, ctx, r));
+    }
+
```

Comment:
Isn't it cleaner to just have a method in the parent class that that throws an `UnsupportedOperationException` instead of having these code paths that shouldn't be invoked?

Reply:
I'm not sure I understand. Each server has a new `processRequest` method that will be used by the new implementations of the RPCs. In a previous comment, you suggested making `processRequest` abstract in `AbstractServer`, so here we provide the implementation.